realtime/redisSubscriber.js

```diff
@@ -24,11 +26,30 @@ module.exports = (io) => {
     // message object to be sent to the clients
     const mssgObj = JSON.parse(mssgStr);
     const key = Object.keys(mssgObj)[0];
+    const parsedObj = rtUtils.parseObject(mssgObj[key], key);
+    if (featureToggles.isFeatureEnabled('publishPartialSample') &&
+    rtUtils.isThisSample(parsedObj)) {
+      const useSampleStore =
+        featureToggles.isFeatureEnabled('enableRedisSampleStore');
 
-    /*
-     * pass on the message received through the redis subscriber to the socket
-     * io emitter to send data to the browser clients.
-     */
-    emitter(io, key, mssgObj);
+      // assign the subject db model if sampleStore is not enabled
+      const subjectModel =
+        useSampleStore ? undefined :
+          require('../db/index').Subject; // eslint-disable-line global-require
+      rtUtils.attachAspectSubject(parsedObj, useSampleStore, subjectModel)
```

Comment:
1. Yes, we can do it that way too but that will have some if statements with 5 lines of code VS this. If we think using "if " statement is more readable, I can change it. 
2. aspectModel is not being passed here. attachAspectSubject takes in aspectModel because it is used by the publisher too and sample post and sample deleteReleatedLinks call the publisher by passing it the aspectModel.